pkg/client/listers/openebs/v1alpha1/castemplate.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2017 The OpenEBS Authors
```

Comment:
nit : 2018 ?

Reply:
As we are adding new files in 2018.

Reply:
this is a old stuff .. moved into this new file... however will change ... thanks